### PR TITLE
Nicolas - SF Automation - Master LPD-35875

### PR DESCRIPTION
**Pull request by MicaelleSilvaa**
Ticket: https://liferay.atlassian.net/browse/LPD-35875

### CODE REVIEW

**Comment by MicaelleSilvaa** [comment]
ci:test:sf

**Comment by MicaelleSilvaa** [comment]
ci:test:hotel

**Comment by MicaelleSilvaa** [comment]
ci:test:sf

File: modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/expected/upgrade/UpgradeCatchAllCheck.testjava [hunk 1]
```diff
@@ -1138,7 +1138,7 @@ public class UpgradeCatchAllCheck implements ConfigurationBeanDeclaration {
 	}
 
 	public void method(
-		String searchEngineId, long companyId, Collection<Document> documents,
+		String searchEngineId, long companyId, Document documents,
```
Comment (nicolas-sss) [inline modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/expected/upgrade/UpgradeCatchAllCheck.testjava]: should be document

**Review by nicolas-sss — CHANGES_REQUESTED** [review]

File: modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/expected/upgrade/UpgradeCatchAllCheck.testjava (hunk 1 above)
Comment (MicaelleSilvaa) [reply modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/expected/upgrade/UpgradeCatchAllCheck.testjava]: Resolved :)

**Comment by MicaelleSilvaa** [comment]
ci:test:sf

**Comment by MicaelleSilvaa** [comment]
ci:test:hotel

**Comment by nicolas-sss** [comment]
ci:test:sf

**Comment by MicaelleSilvaa** [comment]
forward to brian: https://github.com/brianchandotcom/liferay-portal/pull/154053

**Pull request closed without merging**